doc/fuzzing.md

```diff
@@ -7,9 +7,8 @@ To quickly get started fuzzing Bitcoin Core using [libFuzzer](https://llvm.org/d
 ```sh
 $ git clone https://github.com/bitcoin/bitcoin
 $ cd bitcoin/
+# macOS users: make sure to read ["macOS hints for libFuzzer"](#macos-hints-for-libfuzzer)
 $ cmake --preset=libfuzzer
-# macOS users: If you have problem with this step then make sure to read "macOS hints for
```

Comment:
I liked the old version better where it's only there in case you run into problems. There is no need to send people there preemptively. For example maybe the issues are fixed with the next llvm version or so. Then we would save users time with the old comment.

Reply:
I don't think we can run these instructions on a mac currently - can we? I always had to jump to the mac section.

> maybe the issues are fixed with the next llvm version or so

You mean we will be able to remove the special mac section in that case? We can remove the comment as well in that case, but I think we always have to jump over if on a mac - please correct me if I'm wrong.

Reply:
I am here commenting in this issue because I ran into the problem. Macos users that don't have this problem will probably not even look at this issue. So could be that the people that have the problem are in the minority.

Reply:
We're continually updating this doc and for now we're adding more and more to the mac section - it's not getting any simpler for now, seems we still need the mac section for now. And when that changes, we'll update the doc.